### PR TITLE
Adds a Getting Started page.

### PR DESCRIPTION
**Pull request by francocipollone**
Related to #101

### CODE REVIEW

**Comment by francocipollone** [comment]
@tfoote @agalbachicar 
I'd like to get some feedback about this Getting Started page before proceeding to some guidelines with the rule-related api.  See [98f7187](https://github.com/maliput/maliput_documentation/pull/102/commits/98f71876869789f4368de78fcc25ccd0cd764ef2)

I took the liberty of manually generating the online documentation so it can be better checked.
See https://maliput.readthedocs.io/en/latest/index.html and look for the Getting Started page.

File: docs/getting_started.rst [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+The Basics
+==========
+
+Loading a RoadNetwork
+---------------------
+
+The `maliput::api::RoadNetwork` is the main entry point to the library. It is used to load a road network from a backend.
+Once the it is loaded, you can access to the road network's element, such as:
+
+* `maliput::api::RoadGeometry`: It provides several geometry queries making focus on the road surface and semantic of lanes.
```
Comment (tfoote) [inline docs/getting_started.rst:33]: Linking to the API docs would be valuable, aka https://maliput.readthedocs.io/en/latest/html/deps/maliput/html/classmaliput_1_1api_1_1_road_geometry.html for this one.

**Review by tfoote — COMMENTED** [review]
It looks like a good structure to me. This is a guiding point but also I'd suggest giving links for deeper investigations.

File: docs/getting_started.rst (hunk 1 above)
Comment (francocipollone) [reply docs/getting_started.rst]: Done.

**Comment by francocipollone** [comment]
I'd like to address the "Advanced" part in a follow-up PR so we can at least provide a getting started page right away.

File: docs/getting_started.rst [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+This page will help you out getting started with the library.
+
+
+.. warning::
+  Please go to the :ref:`installation_label` page before you start.
+
+
+As explained in the :ref:`maliput_overview_label` page, `maliput` package is an API which implementation is provided by a backend. At the moment there are three different implementations:
+`maliput_dragway <https://github.com/maliput/maliput_dragway>`_, `maliput_multilane <https://github.com/maliput/maliput_multilane>`_, `maliput_malidrive <https://github.com/maliput/maliput_malidrive>`_.
```
Comment (agalbachicar) [inline docs/getting_started.rst:19]:
```suggestion
`maliput_dragway <https://github.com/maliput/maliput_dragway>`_, `maliput_multilane <https://github.com/maliput/maliput_multilane>`_, and `maliput_malidrive <https://github.com/maliput/maliput_malidrive>`_.
```

File: docs/getting_started.rst [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+
+The `maliput::api::RoadNetwork` is the main entry point to the library. It is used to load a road network from a backend.
+Once the it is loaded, you can access to the road network's element, such as:
+
+* `maliput::api::RoadGeometry`_: It provides several geometry queries making focus on the road surface and semantic of lanes.
+* `maliput::api::IntersectionBook`_: Provides information about the intersections in the road network.
+* `maliput::api::rules::TrafficLightBook`_: Related to traffic lights in the road network.
+* `maliput::api::rules::RuleRegistry`_: Related to types of rules that are registered for the road network.
+* `maliput::api::rules::RangeValueRule`_: Related to rules indicating a range of validity for each rule's state.
+* `maliput::api::rules::DiscreteValueRule`_: Related to rules indicating a discrete value for each rule's state.
```
Comment (agalbachicar) [inline docs/getting_started.rst:38]:
```suggestion
* `maliput::api::rules::DiscreteValueRule`_: Related to rules indicating a discrete value for each rule's state (e.g. right of way).
```

File: docs/getting_started.rst [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+There are two ways of loading a RoadNetwork:
+- Using the maliput backend's entry point for loading a RoadNetwork.
+- Using the `Maliput Plugin Architecture <html/deps/maliput/html/maliput_plugin_architecture.html>`_ where the maliput backends/implementations are loaded in runtime.
+
+Let's focus on the first way of loading a road network.
+
+Load a maliput_malidrive RoadNetwork
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+First of all make sure that targets are correctly linked.
```
Comment (agalbachicar) [inline docs/getting_started.rst:64]:
```suggestion
Make sure that targets are correctly linked.
```

Comment (agalbachicar) [reply docs/getting_started.rst:64]: I'm curious about the suggestion. Can you expand?

File: docs/getting_started.rst [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+
+
+.. code-block:: cpp
+  :linenos:
+
+  std::map<std::string, std::string> road_network_configuration;
+  road_network_configuration.emplace("opendrive_file", "<path_to_xodr_file>");
+  auto road_network = malidrive::loader::Load<malidrive::builder::RoadNetworkBuilder>(road_network_configuration);
+
+There are several parameters that can be passed to the malidrive loader. In this case, `opendrive_file` parameters is suggested as the `maliput_malidrive` relies on the OpenDRIVE standard for describing road networks. You can check all the `maliput_malidrive`'s parameters at
```
Comment (agalbachicar) [inline docs/getting_started.rst:89]:
```suggestion
There are several parameters that can be passed to the `maliput_malidrive` loader. In this case, `opendrive_file` parameters is suggested as the `maliput_malidrive` relies on the OpenDRIVE standard for describing road networks. You can check all the `maliput_malidrive`'s parameters at 
```

File: docs/getting_started.rst [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+
+
+For a complete maliput api reference please visit: `maliput::api <html/deps/maliput/html/namespacemaliput_1_1api.html>`_
+
+
+
+Loading a RoadNetwork via Maliput Plugin Architecture
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+TODO
```
Comment (agalbachicar) [inline docs/getting_started.rst:136]: Can we replace with an issue ID?

Comment (agalbachicar) [reply docs/getting_started.rst:136]: Same comment in other TODOs in this file.

File: docs/getting_started.rst [hunk 7]
```diff
@@ -0,0 +1,196 @@
...
+PhaseRingBook
+-------------
+
+TODO
+
+
+
+
+
+
```
Comment (agalbachicar) [inline docs/getting_started.rst:196]:
```suggestion

```

File: docs/getting_started.rst (hunk 4 above)
Comment (francocipollone) [reply docs/getting_started.rst:64]: Just commenting about the next cmake code block. Maybe rephrasing to
`"Link to maliput's libraries:"`
is clearer

File: docs/getting_started.rst (hunk 6 above)
Comment (francocipollone) [reply docs/getting_started.rst]: Done.

File: docs/getting_started.rst [hunk 8]
```diff
@@ -0,0 +1,196 @@
...
+The `maliput_malidrive <https://github.com/maliput/maliput_malidrive>`_ implementation is the one that provides more feature support, so it is the recommended choice.
+
+
+The Basics
+==========
+
+Loading a RoadNetwork
+---------------------
+
+The `maliput::api::RoadNetwork` is the main entry point to the library. It is used to load a road network from a backend.
```
[the remark this answers is not in the document]
Comment (francocipollone) [reply docs/getting_started.rst]: Rephrased

**Review by tfoote — APPROVED** [review]

**Comment by stonier** [comment]
Late to the discussion sorry, but looking good. Be nice if some of those introductory examples are code (esp. the TODOs).

**Pull request merged**